index.js

```diff
@@ -0,0 +1,114 @@
+import * as core from "@actions/core";
+import fetch from "node-fetch";
+
+const NETLIFY_BASE_URL = `https://api.netlify.com/api/v1/`;
+const READY_STATES = ["ready", "current"];
+const WAIT_TIMEOUT = 60 * 15; // 15 min
+const WAIT_INCREMENT = 15; // seconds
+
+const netlifyFetch = async (netlifyToken, url) => {
+  return await fetch(NETLIFY_BASE_URL + url, {
+    headers: {
+      Authorization: `Bearer ${netlifyToken}`,
+    },
+  }).then((response) => response.json());
+};
```

Comment:
this might not be doing what is expected, what are we returning here? `netlifyFetch` should be a function which calls `fetch()` and then returns the response json (as a promise), right? then i think this is the syntax:

```suggestion
const netlifyFetch = async (netlifyToken, url) => {
  const response = await fetch(NETLIFY_BASE_URL + url, {
    headers: {
      Authorization: `Bearer ${netlifyToken}`,
    },
  });
  return await response.json();
};
```

Reply:
Done.